### DISCUSSION TITLE
Migration to 2.0.0

### DISCUSSION THREAD

**Discussion by chkpnt** [eclipse-serializer/serializer#149] [Q&A] [answered]
I've just seen you've published Eclipse Serializer v2.0.0.  🎉

From the changes mentioned in the [release notes](https://github.com/eclipse-serializer/serializer/releases/tag/2.0.0), I do not think there is anything to regard when migrating from 1.x. Correct?

**Comment by zdenek-jonas** [comment, marked as answer]
No migration is needed; just update your dependency.
